test/Dialect/SPIRV/control-flow-ops.mlir

```diff
@@ -144,6 +144,119 @@ func @weights_cannot_both_be_zero() -> () {
 
 // -----
 
+//===----------------------------------------------------------------------===//
+// spv.FunctionCall
+//===----------------------------------------------------------------------===//
+
+spv.module "Logical" "GLSL450" {
+  func @fmain(%arg0 : vector<4xf32>, %arg1 : vector<4xf32>, %arg2 : i32) -> () {
+    // CHECK: {{%.*}} = spv.FunctionCall @f_0({{%.*}}, {{%.*}}) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
+    %0 = spv.FunctionCall @f_0(%arg0, %arg1) : (vector<4xf32>, vector<4xf32>) -> vector<4xf32>
+    // CHECK: spv.FunctionCall @f_1({{%.*}}, {{%.*}}) : (vector<4xf32>, vector<4xf32>) -> ()
+    spv.FunctionCall @f_1(%0, %arg1) : (vector<4xf32>, vector<4xf32>) ->  ()
+    // CHECK: spv.FunctionCall @f_2() : () -> ()
+    spv.FunctionCall @f_2() : () -> ()
+    // CHECK: {{%.*}} = spv.FunctionCall @f_4({{%.*}}) : (i32) -> i32
+    %1 = spv.FunctionCall @f_4(%arg2) : (i32) -> i32
+    // CHECK: {{%.*}} = spv.FunctionCall @f_4({{%.*}}) : (i32) -> i32
+    %2 = spv.FunctionCall @f_4(%1) : (i32) -> i32
+    spv.Return
+  }
+
+  func @f_0(%arg0 : vector<4xf32>, %arg1 : vector<4xf32>) -> (vector<4xf32>) {
+    spv.ReturnValue %arg0 : vector<4xf32>
+  }
+
+  func @f_1(%arg0 : vector<4xf32>, %arg1 : vector<4xf32>) -> () {
+    spv.Return
+  }
+
+  func @f_2() -> () {
+    spv.Return
+  }
+
+  func @f_3(%arg0 : !spv.struct<!spv.array<4xf32>>) -> (!spv.struct<!spv.array<4xf32>>) {
+    // CHEKC: {{%.*}} = spv.FunctionCall @f_3({{%.*}}) : (!spv.struct<!spv.array<4xf32>>) -> !spv.struct<!spv.array<4xf32>>
```

Comment:
Typo: CHECK.

Do we need this case? Seems already covered by fmain. (I mean we don't check new stuff here right?)